UnityGLTF/Assets/UnityGLTF/Runtime/Plugins/GLTFSerialization/Schema/MeshPrimitive.cs

```diff
@@ -321,10 +319,12 @@ public override void Serialize(JsonWriter writer)
 
 	public static class SemanticProperties
 	{
-		public const string POSITION = "POSITION";
+		public const string POSITION = "POSITION"; 
 		public const string NORMAL = "NORMAL";
+		public const string JOINT = "JOINT";
+		public const string WEIGHT = "WEIGHT";
 		public const string TANGENT = "TANGENT";
-		public const string INDICES = "INDICES";
+		public const string INDICES = "INDICIES";
```

Comment:
This seems odd. The spec itself spells it `indices`: https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#indices-and-names

Why's this being re-spelled?

Reply:
Fixed a while ago, thanks!